src/resources/authoring/gv2.json

```diff
@@ -54,7 +54,7 @@
       "targetDrakes": [{},{}],
       "userChangeableGenes": "wings, forelimbs, hindlimbs"
     },
-    "eggSortingWings": {
+    "basicEggSortingWings": {
```

Comment:
They haven't...let me ping her and ask her to fill them out in the Geniverse development master